docs/excel/custom-functions-overview.md

```diff
@@ -266,6 +267,31 @@ In some situations, you may need to cancel the execution of a streaming custom f
 
 To enable the ability to cancel a function, you must implement a cancellation handler within the JavaScript function and specify the property `"cancelable": true` within the `options` object in the JSON metadata that describes the function. The code samples in the previous section of this article provide an example of these techniques.
 
+## Declaring a volatile function
+
+[Volatile functions](https://docs.microsoft.com/office/client-developer/excel/excel-recalculation#volatile-and-non-volatile-functions) are functions in which the value changes from moment to moment, even if none of the function's arguments have changed. These functions recalculate every time Excel recalculates. For example, imagine a cell that calls the function `=NOW`. Every time `=NOW` is called, it will give you the current date and time, automatically.
+
+There are a handful of built in volatile functions in Excel, such as `=RAND` and `=TODAY`. You can find [a comprehensive list of Excel’s volatile functions here](https://docs.microsoft.com/en-us/office/client-developer/excel/excel-recalculation#volatile-and-non-volatile-functions).  
+  
+Custom functions allow you to create your own volatile functions, which you may want to use to deal with dates, times, random numbers, and for modelling purposes. For example, Monte Carlo simulations require generation of random inputs to determine an optimal solution.  
+  
+To declare a function volatile, you'll add `"volatile": true` under the `options` property in the function's JSON file, as shown in the following code sample. Note that a function cannot be marked both `"streaming": true` and `"volatile": true`; in the case where both are marked `true` the volatile option will be ignored.  
```

Comment:
Suggest that you rephrase as follows -> 

To declare a function volatile, add `"volatile": true` within the `options` object for the function in the JSON metadata file, as shown in the following code sample. Note that a function cannot be marked both `"streaming": true` and `"volatile": true`; if both properties are set to `true`, the volatile option will be ignored.

Reply:
Random question: when do we usually use the declarative over the "you will see" kind of instruction? I have seen both in docs and just wondering if we have some general guidelines about using declarative more frequently.